wasm/integration-test/sqlite/src/lib.rs

```diff
@@ -0,0 +1,319 @@
+//! Hermes SQLite module integration test with WASM runtime.
+//! Generate `hermes.rs` with `earthly +gen-bindings` before writing the test.
+
+// Allow everything since this is generated code.
+#[allow(clippy::all, unused)]
+mod hermes;
+
+use hermes::{
+    exports::hermes::integration_test::event::TestResult,
+    hermes::{
+        cardano::api::{BlockSrc, CardanoBlock, CardanoBlockchainId, CardanoTxn},
+        cron::api::CronTagged,
+        kv_store::api::KvValues,
+        sqlite,
+    },
+    wasi::http::types::{IncomingRequest, ResponseOutparam},
+};
+
+struct TestItem {
+    name: &'static str,
+    executor: fn() -> bool,
+}
+
+const TESTS: &[TestItem] = &[
+    TestItem {
+        name: "open-database-persistent-simple",
+        executor: || {
```

Comment:
These inline functions are quite long, it would be nicer if they were actual functions we just reference here by name.

Reply:
Also, put the functions in another file, as this one is getting long already.